supabase-cli: 2.23.7 -> 2.33.9

stephenstubbs opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done
Bug fixes and upgrades see https://github.com/supabase/cli/releases
cc: @Gerschtli @kashw2 @r-ryantm
<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

kashw2 reviewed (changes requested):
Please update the `hash` field inside of the `src` set. The `vendorHash` may also require an update

stephenstubbs commented:
@kashw2 it seems to work for me when I install from this branch to test which is why I didn't change them to lib.fakeHash to see what I need to update them too. I vaguely remember it warning me that the hashes were incorrect when I updated a package before and then I updated them but it doesn't seem to be the case with this one.

kashw2 commented:
For me the hash I get is `sha256-bH/ifvyeZ57GRBzqa9efMKBs5/6U5O10gwpOsDt1Mdc=` and the vendorHash being `sha256-8gqUlnfyQ41IRa65jBs8SO9bLIE+Rh0iypZt5Ll3wtg=`.

With 10 feature updates as per semver I'd expect that there are changes to the program that would contribute to both a hash and vendorHash change

kashw2 reviewed (commented):
Please squash and list commits into each logical change such as:

`supabase-cli: 2.33.7 -> 2.33.9`
`supabase-cli: add subPackages, removed unnecessary command`

or something similar. Other than that, this LGTM

stephenstubbs committed:
supabase-cli: 2.33.7 -> 2.33.9

stephenstubbs committed:
supabase-cli: add subPackages, removed unnecessary command

daviaaze reviewed (approved):
Tested and seems to be working fine

stephenstubbs commented:
Thank you. Is there anything left for me to do @daviaaze or can this get merged in?

Sigmanificient commented:
@stephenstubbs Someone with commit bits need to press the merge button

MattSturgeon commented:
I'm assuming the CI failure was an ofborg issue rather than a packaging issue?

https://github.com/NixOS/nixpkgs/pull/431891/checks?check_run_id=48016238422

Defelo commented:
@MattSturgeon The `nixpkgs` flake registry entry points to the `nixpkgs-unstable` branch (at least on gha, see output of `nix flake metadata nixpkgs`), so you just built the old version of this package. The new version doesn't actually build because only the FOD hashes have been updated, but not the `version`. See https://github.com/Defelo/nixpkgs-review-gha/actions/runs/17000589882
